lib/apartment/tenant.rb

```diff
@@ -25,9 +25,7 @@ def init
     #
     def adapter
       Thread.current[:apartment_adapter] ||= begin
-        config = Apartment.default_tenant
-
-        adapter_name = "#{config[:adapter]}_adapter"
+        adapter_name = "postgresql_adapter"
```

Comment:
What was wrong with the old code here? It looks to be more flexible?

Reply:
The old code assumes that the default tenant is a hash, instead of a tenant name. We could pull up the config for the default tenant, but I went with this since we only ever need the postgresql adapter.